src/large_pages/node_large_page.cc

```diff
@@ -263,18 +263,17 @@ bool IsTransparentHugePagesEnabled() {
   // File format reference:
   // https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/mm/huge_memory.c?id=13391c60da3308ed9980de0168f74cce6c62ac1d#n163
   ifs.open("/sys/kernel/mm/transparent_hugepage/enabled");
```

Comment:
```suggestion
  std::ifstream ifs("/sys/kernel/mm/transparent_hugepage/enabled");
```
nit: to construct the object and open the file at once

Reply:
It doesn't look like there are any other read-only stream file `open(...)` calls in the codebase currently, so to some extent we're arguably creating a pattern for the codebase here that may be followed by others.

Although opening files in read-only mode is the default per the `const char* filename` overloaded constructor ( https://en.cppreference.com/w/cpp/io/basic_ifstream/basic_ifstream ), what do you think about the read-only behaviour explicit by specifying `std::ios::in` as the second argument?

Reply:
This suggestion is now included, as is a further adjustment to open the file with read-only access explicitly.

Reply:
I agree, it is better to explicitly use `std::ios::in` to make it clearer. Thanks for adding it in. :)